ui/perfherder/constants.js

```diff
@@ -1,2 +1,8 @@
 export const tValueCareMin = 3; // Anything below this is "low" in confidence
 export const tValueConfidence = 5; // Anything above this is "high" in confidence
+
+// Backend server endpoints
+export const endpoints = Object.freeze({
```

Comment:
I think this is unnecessary. You can declare them as `const`s and call it a day.